{
  "id": "oval:org.opensuse.security:def:201914846",
  "version": "1",
  "class": "vulnerability",
  "metadata": {
    "Title": "CVE-2019-14846",
    "affected": {
      "family": "unix",
      "Platform": [
        "SUSE Linux Enterprise Server 11 SP3-TERADATA"
      ]
    },
    "reference": [
      {
        "ref_id": "Mitre CVE-2019-14846",
        "ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-14846",
        "source": "CVE"
      },
      {
        "ref_id": "SUSE CVE-2019-14846",
        "ref_url": "https://www.suse.com/security/cve/CVE-2019-14846",
        "source": "SUSE CVE"
      },
      {
        "ref_id": "SUSE-SU-2020:3309-1",
        "ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-November/007763.html",
        "source": "SUSE-SU"
      },
      {
        "ref_id": "openSUSE-SU-2020:0513-1",
        "ref_url": "https://lists.opensuse.org/opensuse-security-announce/2020-04/msg00021.html",
        "source": "SUSE-SU"
      },
      {
        "ref_id": "openSUSE-SU-2020:0523-1",
        "ref_url": "https://lists.opensuse.org/opensuse-security-announce/2020-04/msg00026.html",
        "source": "SUSE-SU"
      },
      {
        "ref_id": "openSUSE-SU-2022:0081-1",
        "ref_url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/D7KK2SNPNAB353QA6BU4SNJDQ3FXZOY5/",
        "source": "SUSE-SU"
      }
    ],
    "Description": "\n    In Ansible, all Ansible Engine versions up to ansible-engine 2.8.5, ansible-engine 2.7.13, ansible-engine 2.6.19, were logging at the DEBUG level which lead to a disclosure of credentials if a plugin used a library that logged credentials at the DEBUG level. This flaw does not affect Ansible modules, as those are executed in a separate process.\n    ",
    "advisory": {
      "from": "security@suse.de",
      "Severity": "Moderate",
      "cve": {
        "text": "CVE-2019-14846",
        "href": "https://www.suse.com/security/cve/CVE-2019-14846/",
        "impact": "moderate",
        "cvss3": "2.3/CVSS:3.0/AV:L/AC:L/PR:H/UI:N/S:U/C:L/I:N/A:N"
      },
      "bugzilla": [
        {
          "text": "SUSE bug 1153452",
          "href": "https://bugzilla.suse.com/1153452"
        }
      ],
      "issued": {
        "date": "2022-03-01"
      },
      "updated": {
        "date": "2022-03-17"
      },
      "affected_cpe_list": {
        "Cpe": [
          "cpe:/o:suse:sles:11:sp3:teradata"
        ]
      }
    }
  },
  "criteria": {
    "operator": "AND",
    "criterias": [
      {
        "operator": "OR",
        "criterions": [
          {
            "test_ref": "oval:org.opensuse.security:tst:2009577265",
            "comment": "ansible-2.9.14-11.14.1 is installed"
          },
          {
            "test_ref": "oval:org.opensuse.security:tst:2009577262",
            "comment": "python-coverage-3.6-0.11.2.1 is installed"
          },
          {
            "test_ref": "oval:org.opensuse.security:tst:2009577263",
            "comment": "python-passlib-1.6.1-0.3.2.1 is installed"
          }
        ]
      }
    ],
    "criterions": [
      {
        "test_ref": "oval:org.opensuse.security:tst:2009077550",
        "comment": "SUSE Linux Enterprise Server 11 SP3-TERADATA is installed"
      }
    ]
  }
}
